docs/guides/jsdom.md

```diff
@@ -11,6 +11,33 @@ gets run *before* React's code is run.
 
 As a result, a standalone script like the one below is generally a good approach:
 
+`jsdom v10~`:
+
+```js
+/* setup.js */
+
+var { JSDOM } = require('jsdom');
+var jsdom = new JSDOM('<!doctype html><html><body></body></html>');
+var { window } = jsdom;
+
+global.window = window;
+global.document = window.document;
+
+Object.keys(window).forEach((property) => {
```

Comment:
Thanks for the advise, that way would be better!

Fixed with: https://github.com/airbnb/enzyme/pull/921/commits/57dc37a58ea17a690895b82702a0a983a74b7ebd